### DISCUSSION TITLE
I would like drag & drop support in the side bar

### DISCUSSION THREAD

**Discussion by Limezy** [athensresearch/athens#964] [Feature Requests]
Pretty much everything is in the title 😄

**Comment by solarkraft** [comment]
I don't understand what you mean yet. Do you mean to drag and drop nodes or references to pages into the sidebar? Because rearranging pinned pages already works.

**Reply by Limezy** [reply]
Sorry, my wording was confusing. The drag and drop rearranging indeed works with shortcuts in the **left** sidebar, but it doesn't between side-opened pages or graphs on the **right** sidebar.

**Comment by jefftangx** [comment]
I want this too!

**Comment by shanberg** [comment]
I've assumed this is about the navigation sidebar, but it would also be nice to be able to rearrange items in the reference (right) sidebar too.
